lib/rails_semantic_logger/extensions/active_support/logger.rb

```diff
@@ -4,7 +4,7 @@ module ActiveSupport
   # More hacks to try and stop Rails from being it's own worst enemy.
   class Logger
     class << self
-      undef :logger_outputs_to?, :broadcast
+      undef :logger_outputs_to?
```

Comment:
Should also take into account `def self.broadcast` below. I think this should be

```ruby
# This only applies to Rails <7.1
if defined(:broadcast)
  undef :broadcast
  def broadcast(logger)
    Module.new
  end
end
```

Reply:
Makes sense, updated :)